src/ios/SQLitePlugin.m

```diff
@@ -54,12 +54,18 @@
 }
 #endif
 
+@interface SQLitePlugin()
+@property (nonatomic, readonly, strong) id <OSLoggerProtocol> logger;
```

Comment:
Same as the Android counterpart. The dependency is missing on plugin.xml

Reply:
done ✅

Reply:
I do have some concerns about adding the logger to a public repository. I believe this there is no precedent and should be discussed with the team (literally stop the team for a moment), because once this happens there is basically no going back.